### PR TITLE
Pack labels

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR documents how to add labels and annotations to Add-on Packs.

CleanShot 2023-08-14 at 14 30 17

## Review Changes

💻 [Preview URL](https://deploy-preview-1456--docs-spectrocloud.netlify.app/cluster-profiles/create-add-on-profile#packlabelsandannotations)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/PCP-1661)

### CODE REVIEW

File: content/docs/04-clusters/06-cluster-management/08-taints.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Node Labels and Taints"
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/08-taints.md:2]: @ritawatson this file only has title changes.

File: content/docs/03-cluster-profiles/01.5-create-add-on-profile.md [hunk 2]
```diff
@@ -21,10 +21,57 @@ Add-on cluster profiles offer a range of benefits for workload cluster deploymen
...
+| **Parameter** | **Description** | **Type** |
+| --- | --- | --- |
+| `namespace` | The Namespace that the pack is deployed to. If the Namespace does exists, then Palette will create the Namespace. | string |
+| `additionalNamespaces`| A list of additional Namespaces that Palette will create. | list |
+| `namespaceLabels` | A list of key value pairs that are applied to the Namespace. | list |
+| `namespaceAnnotations` | A list of key value pairs that are applied to the Namespace. | list |
+
+
+
+The following example displays how to specify Namespace labels and annotations for an Add-on Pack, a CSI pack, and a CNI pack. In the example pack YAML configuration, the `wordpress` Namespace is created. An additional Namespace titled `wordpress-storage` is also created. In the parameters sections `namespaceLabels`, and `namespaceAnnotations`, each entry is made up of a key and a value. The key is the name of the target Namespace, and the value is the value of the label or annotation.
```
Comment (ritawatson) [inline content/docs/03-cluster-profiles/01.5-create-add-on-profile.md:39]:
```suggestion
The following example shows how to specify Namespace labels and annotations for an Add-on Pack, a CSI pack, and a CNI pack. In the example pack YAML configuration, the `wordpress` Namespace is created. An additional Namespace titled `wordpress-storage` is also created. In the parameters sections, `namespaceLabels` and `namespaceAnnotations`, each entry has a key and a value. The key is the name of the target Namespace, and the value is the value of the label or annotation.
```

File: content/docs/04-clusters/06-cluster-management/08-taints.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-metaTitle: "Label and Taint the nodes"
-metaDescription: "Apply taints on node pools for appropriate node to pod scheduling."
+title: "Labels and Taints"
+metaTitle: "Labels and Taints"
+metaDescription: "Learn how to apply labels and taints to nodes in a cluster, and how to specify Namespace labels and annotations to Add-on packs, and packs that are for Container Storage Interfaces (CSI) and Container Network Interfaces (CNI) drivers."
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/08-taints.md:4]:
```suggestion
metaDescription: "Learn how to apply labels and taints to nodes in a cluster, and how to specify Namespace labels and annotations to Add-on packs and packs for Container Storage Interface (CSI) and Container Network Interface (CNI) drivers."
```

File: content/docs/04-clusters/06-cluster-management/08-taints.md [hunk 4]
```diff
@@ -40,12 +40,12 @@ Eg: Key = key1;
...
 
 ## Apply Labels to nodes
 
-Labels are specified in an optional field called 'Additional Labels' in the node pool configuration form. The values are specified as 'key:value'. Multiple such labels may be specified. Labels may be specified from the node pool conifguration form initially during cluster provisioning and updated any time by editing a node pool from the 'Nodes' tab of the cluster details page.
+Labels are specified in an optional field called 'Additional Labels' in the node pool configuration form. The values are specified as 'key:value'. Multiple such labels may be specified. Labels may be specified from the node pool conifguration form initially during cluster provisioning and updated any time by editing a node pool from the 'Nodes' tab of the cluster details page.
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/08-taints.md:51]:
```suggestion
Labels are optional and can be specified in the **Additional Labels** field of the node pool configuration form. Specify one or more values as 'key:value'. You can specify labels initially during cluster provisioning and update them any time by editing a node pool from the **Nodes** tab of the cluster details page.
```

**Review by ritawatson — APPROVED** [review]
I offered a few fixes to address.

**Pull request merged**